src/js/plugins/tts/FestivalTTSEngine.js

```diff
@@ -120,4 +104,24 @@ export default class FestivalTTSEngine extends AbstractTTSEngine {
                   + encodeURIComponent(dataString)
                   + '&format=.'+this.audioFormat;
     }
-}
\ No newline at end of file
+
+    /**
+     * @private
+     * Security restrictions require playback to be triggered
+     * by a user click/touch. This intention gets lost in the async calls
+     * on iOS, but, for some reason, if we start the audio here, it works.
+     * See https://stackoverflow.com/questions/12206631/html5-audio-cant-play-through-javascript-unless-triggered-manually-once
+     * @return {PromiseLike}
+     */
+    iOSCaptureUserIntentHack() {
+        let sound = soundManager.createSound({ url: SILENCE_1MS[this.audioFormat] });
+        return new Promise(res => sound.play({onfinish: res}))
+        .then(() => sound.destruct());
+    }
+}
+
+/** Needed to capture the audio context for iOS hack. Generated using Audacity. */
+const SILENCE_1MS = {
```

Comment:
https://github.com/mathiasbynens/small

Smallest possible syntactically valid files of different types

Reply:
no action required

Reply:
I'm afraid it might cause new weirdness to happen (like it might not be able to play it? Trying to open the mp3 file in FF gives a MIME type error). I'll create an issue to investigate compressing them further.

(I do think it needs some length, cause it has to play it to capture the context (I think)).